feature_layers/feature-collection-layer-from-portal/README.metadata.json

```
@@ -0,0 +1,35 @@
+{
+    "category": "Feature layers",
+    "description": "Combine feature tables with different geometries into a single layer.",
+    "ignore": false,
+    "images": [
+        "FeatureCollectionLayer.png"
+    ],
+    "keywords": [
+        "FeatureCollection",
+        "FeatureCollectionLayer",
+        "FeatureCollectionTable",
+        "Feature",
+        "Field",
+        "SimpleFillSymbol",
+        "SimpleLineSymbol",
+        "SimpleMarkerSymbol",
+        "SimpleRenderer"
```

These should be the 'Tags' from the readme:
```suggestion
        "collection", 
        "feature collection", 
        "feature collection layer", 
        "id", 
        "item", 
        "map "notes, 
        "portal"
```